devops/roles/analytics-druid/vars/all_pass.yml

```diff
@@ -0,0 +1,14 @@
+$ANSIBLE_VAULT;1.1;AES256
+61306132653639663365633235626363383037393731346635323635333265653266323936643739
+3934313933396166616237666665303638643139633565610a663636313666363031383234323837
+61666636376330343837343934333162343664353939363639303663303737383338376430313036
+3066633061323533370a666362353263613662313061396536356564643938343833313262323933
+61383738623039393166346437616561323262626635663837633335626465633731313837353037
+66393463613334303866376234623163396237356363653737376539363763633465623537383761
+39373436643231346539393264646332326633623932663633363362373434656332616465376264
+30326366643439336363363065643536646538356162343438333364356530313639393231323664
+37356132313763336234323066616362373565393434326565653064653037393432636633376134
+66393732306437346332356638366334346235616266656266316232396261383638396437303537
+32353266366437613237333231323365316365373865336438623335383164613264626237316233
+35316439356265323961666561656439303533643763643532613136303037396364656137633464
+3439
```

Comment:
if this contains any values, please move it to <inventory_dir>/secrets.yml

Reply:
@rjshrjndrn yes this is being moved to the <inventory_dir>/secrets.yml file.